src/cross_section_measurement/make_control_plots_fromTrees.py

```diff
@@ -39,7 +39,7 @@ def getHistograms( histogram_files,
     # Names of QCD regions to use
     qcd_data_region = ''
     qcd_data_region_electron = 'QCDConversions'
-    qcd_data_region_muon = 'QCD non iso mu+jets 3toInf'
+    qcd_data_region_muon = 'QCD non iso mu+jets 1p5to3'
```

Comment:
Exactly I just changed it while the control region is not good. In the end we should go for the one with the best description of the data